### PR TITLE
initial project folder  #1

### LINKED ISSUE #1: Init project
Refactor Project Structure

### PR DESCRIPTION
**Pull request by 2H1EU8**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (thanhbui27) [inline src/index.html]: I think the index file.html left here doesn't make sense. You can create folders public/index.html

Comment (2H1EU8) [reply src/index.html]: let me fix that

**Comment by thanhbui27** [comment]
I think such a structure should be built
If you want to use Services, you can add

src/
│ ├── controller/
│   │   ├── controller.js
│   ├── model/
│   │   └── model.js
│   └── views/
│        ├── Resources/
 |         │   ├── index.html
 |         │   └── assets/
 |         │       ├── css/
 |         │       └── js/
├── package.json
├── package-lock.json
└── README.md

**Comment by 2H1EU8** [comment]
> I think such a structure should be built
> 
> src/ │ ├── app.js │ ├── controller/ │ │ ├── controller.js │ ├── models/ │ │ └── models.js │ └── views/ │ ├── Resources/ | │ ├── index.html | │ └── assets/ | │ ├── css/ | │ └── js/ ├── package.json ├── package-lock.json └── README.md

Ok i got it

**Comment by thanhbui27** [comment]
pls add config husky and Prettier

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+- Later
+
+## SOURCE CODE
+
+- Clone project : git clone git
+
+## HOW TO RUN
+
+- Dillinger requires Node.js v10+ to run.
```
Comment (thanhbui27) [inline README.md]: Please review this version

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+- Install the dependencies and devDependencies and start the server.
+
+```
+$npm install --save-dev parcel
+$npm start
+Open on web: http://localhost:1234
```
Comment (thanhbui27) [inline README.md:36]: and review this here

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/node_modules
+.eslintrc.json
```
Comment (thanhbui27) [inline .gitignore]: pls fix github action

File: README.md (hunk 3 above)
Comment (2H1EU8) [reply README.md:36]: I think I will update after the project is done

File: .gitignore (hunk 4 above)
Comment (2H1EU8) [reply .gitignore]: Sorry, What u mean ?

Comment (thanhbui27) [reply .gitignore]: image

File: README.md (hunk 3 above)
Comment (thanhbui27) [reply README.md:36]: I think you should update properly as soon as you initialize Pratice

File: README.md (hunk 2 above)
Comment (thanhbui27) [inline README.md]: Currently node JS is version 18 but you still use version 10?

**Comment by thanhbui27** [comment]
folder controller where ?

**Comment by thanhbui27** [comment]
Model without 's'

File: .eslintcache
Comment (baonguyen26) [inline .eslintcache:1]: pls ignore this file

File: README.md (hunk 3 above)
Comment (baonguyen26) [reply README.md:36]: Pls update this file before start coding

File: src/model/book.model.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    this.title = title;
+    this.author = author;
+    this.category = category;
+    this.status = status;
+    this.number = number;
+    this.price = price;
+    this.check = check;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```
Comment (baonguyen26) [inline src/model/book.model.js]: Why do you add this line?

File: src/resources/index.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>H1 Library App</title>
+</head>
+<body>
+    <div class="root">
+        
+    </div>
+
+    <script src="../views/book.view.js"></script>
```
Comment (baonguyen26) [inline src/resources/index.html]: Should embed only app

**Review by baonguyen26 — CHANGES_REQUESTED** [review]
Pls check the end line of file, don't push with github error

File: src/model/book.model.js (hunk 5 above)
Comment (2H1EU8) [reply src/model/book.model.js:16]: image
if i don't add this line, eslint will show an error

File: src/model/book.model.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (thanhbui27) [inline src/model/book.model.js]: Pls remove this

**Pull request merged**